pytorch_blade/src/compiler/jit/fusion.cpp

```diff
@@ -125,5 +125,16 @@ torch::TypePtr get_list_tensor_type() {
   return list_type;
 }
 
+torch::TypePtr tensor_type_from_scalar(const c10::Type& typ) {
```

Comment:
Nit: I prefer `create_tensor_type_from_scalar_type`，also it seems duplicate to https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/jit_type.h#L587

Reply:
Renamed ~ 
Actually this piece of code taken from: 
except return nullptr on failure.

https://github.com/pytorch/pytorch/blob/75ad6fea66bc2f0ff0f58d83a1b198549034c069/aten/src/ATen/core/jit_type.h#L1575-L1586